server/server.js

```diff
@@ -77,9 +77,9 @@ module.exports = port => {
             currentUsers.push(user);
             gameLogic.joinGame(user);
             emitGameState();
-            res.sendStatus(200);
+            res.status(200).json({message: "OK"});
```

Comment:
I might be wrong but I'm not sure you need to specify the 200 response here. Although doing it explicitly isn't a bad thing.

I don't think you need to send the message OK since that's what the 200 code means.